Core/clim-basic/recording.lisp

```diff
@@ -1515,37 +1515,44 @@ were added."
                     (eql (slot-value record 'filled) filled))))
 ;;;; Patterns
 
-;;; The Spec says that "transformation only affects the position at
-;;; which the pattern is drawn, not the pattern itself"
-(def-grecording draw-pattern (() pattern x y) ()
-  (let ((width (pattern-width pattern))
-        (height (pattern-height pattern))
-        (transform (medium-transformation medium)))
-    (setf (values x y) (transform-position transform x y))
-    (values x y (+ x width) (+ y height))))
+(defclass drawing-transform-mixin ()
```

Comment:
since it has transformed X and Y values, better name would be `position-transform-mixin`.